syntax = "proto3";
import public "Backup.proto";
import public "BasicMsg.proto";
import public "TaskMsg.proto";

package msg;

enum ProtoType {
  UNKNOWN = 0;
  BASIC_MSG = 1;
  BACKUP = 2;
  TASK_MSG = 3;
}

message MsgToSend {
  BasicMsg basicMsg = 1;
  Backup backup = 2;
  TaskMsg task = 3;
}
